app/views/application/_header.html.erb

```diff
@@ -1,12 +1,5 @@
 <div class="header">
   <%= link_to "Apply", apply_path, class: "header-tab header-button" %>
-
-  <% if !current_user %>
-    <!-- Temporary log in link -->
-    <%= link_to "Log in", new_admin_session_path, class: "header-tab" %>
-  <% else %>
-    <%= link_to "Log out", destroy_user_session_path, class: "header-tab", method: :delete %>
-  <% end %>
```

Comment:
Yup haha. Also cause the HTTP verb has to be delete so typing in the URL in the browser doesn't work haha. This logout link logs out for all types of users, including students and NPOs, so let's keep it.